app/assets/stylesheets/views/_completed-transaction.scss

```diff
@@ -1,5 +1,5 @@
 .transaction-done {
-  #organ-donor-registration-promotion {
+  #organ-donor-registration-promotion, #mot-reminder-promotion {
```

Comment:
I wonder if we should make this a “promotion” class rather than using IDs in the CSS?

Reply:
I guess it depends on how likely we are to add more in the future.

Reply:
Yeah, I can just see this comment coming up every time we add a new one and it might be worth doing it now 😄

Reply:
Thanks @thomasleese , I've updated this now. I'll change the other promotions to classes rather than ID's in a separate PR.

Reply:
Ah, I actually meant to call the class `promotion` and then on https://github.com/alphagov/frontend/pull/1488/files#diff-6e10a2c10820051cc48f8764b75ff654R13 and the new MOT reminder `div`, changing it to `id="..." class="promotion"`

Reply:
Ah right sorry! I've changed this now.